src/components/ReactSlider/ReactSlider.md

```diff
@@ -250,3 +250,44 @@ const ResizableSlider = () => {
 
 <ResizableSlider />
 ```
+
+Single slider, applying `ariaLabelledby` to establish association with a label
+
+```jsx
+<div>
+    <label id="slider-label" htmlFor="slider">
+        React Slider example
+    </label>
+    <ReactSlider
+        ariaLabelledby="slider-label"
+        className="horizontal-slider"
+        thumbClassName="example-thumb"
+        trackClassName="example-track"
+        renderThumb={(props, state) => <div {...props}>{state.valueNow}</div>}
+    />
+</div>
+```
+
+Double slider, applying `ariaLabelledby` as an array to multiple thumb labels
+
+```jsx
+<div>
+    <label id="first-slider-label" htmlFor="slider">
+        Start slider label
+    </label>
+    <label id="second-slider-label" htmlFor="slider">
```

Comment:
These `<label>`s have an `htmlFor` that currently point to a non-existent id `"slider"`. I think since we are using `aria-labelledby`, you can omit `htmlFor` in this case.

Reply:
Good spot 🤦